libs/api/domains/education/src/lib/models/course.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { CourseCompetence } from './competence.model'
+import { GradeCategory } from './gradeCategory.model'
+import { Grade } from './grade.model'
+
+@ObjectType('EducationCompulsorySchoolCourse')
+export class Course {
+  @Field() //t.d. "Enska"
```

Comment:
Sorry for the nitpick, but I think the coding standard is to keep things in English.
```suggestion
  @Field() //f.x. "Enska"
```

Reply:
good eye

Reply:
removed